core/src/main/java/greencity/controller/EcoNewsController.java

```diff
@@ -329,7 +331,8 @@ public ResponseEntity<Long> findAmountOfPublishedNews(@RequestParam Long userId)
     @ApiResponses(value = {
         @ApiResponse(code = 200, message = HttpStatuses.OK),
         @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
-        @ApiResponse(code = 401, message = HttpStatuses.UNAUTHORIZED)
+        @ApiResponse(code = 401, message = HttpStatuses.UNAUTHORIZED),
+        @ApiResponse(code = 404, message = HttpStatuses.NOT_FOUND)
```

Comment:
Is it expected to throw 404?

Reply:
Yes. When you input wrong ID(like -53) - response is 404, but it wasn't in documentation. So, I added this response.